.github/workflows/pr_prover.yaml

```diff
@@ -0,0 +1,87 @@
+name: Prover
+on:
+  push:
+    branches: ["main"]
+  merge_group:
```

Comment:
same

Reply:
I removed all `merge_group:` lines here [c9c0252](https://github.com/lambdaclass/ethrex/pull/2291/commits/c9c025208fbe7895696742d6eafcde61c000b52d), I don't think we were really using them for anything